docs/development.txt

```diff
@@ -0,0 +1,82 @@
+
+=== Branch Naming Conventions
+
+All branch names shall be prepended with `Issue_#_`.
+
+[[pull-requests]]
+== Pull Requests
+
+Most changes to JanusGraph must be reviewed before they are committed (RTC - review-then-commit).
+This workflow is performed using Github's pull request process. In instances where the committer deems the full process unnecessary, commit-then-review (CTR) may be employed.
+This should be used infrequently and only for extremely trivial changes.
+Changes of this sort may include:
+
+* Documentation typo or small documentation addition
+* Addition of a new test case
+
+Any commit that is made following CTR shall include the fact that it is a CTR in the commit comments.
+The committer should also send a message to https://groups.google.com/forum/#!forum/janusgraph-dev[janusgraph-dev] to notify other project members of the CTR.
+If another committer responds with a -1, the commit should be rolled back and the formal RTC process should be followed.
+
+Users wishing to contribute to JanusGraph should fork the https://github.com/janusgraph/janusgraph[JanusGraph] repository and then submit pull requests using the Github pull request process.
+Every pull request must be associated with an existing issue.
+Be sure to include the relevant issue number in the title of your pull request and a description of what test suites were run to validate the pull request.
+
+Pull requests commit messages should clearly describe what work was accomplished.
+Intermediate work-in-progress commits should be squashed prior to pull request submittal.
+
+Pull requests must be reviewed before being merged.
+The JanusGraph project uses the https://help.github.com/articles/about-pull-request-reviews/[Github review process] to review pull requests.
+Non-committers are free, and encouraged to review pull requests.
+Their review will be non-binding, but can be taken into consideration and are still very valuable community input.
+The following rules apply to the voting process.
+
+* Two committer approvals are required merge a pull request
+* One or more -1 votes within a change request will veto the pull request until the noted issue(s) are addressed and the -1 vote is withdrawn
+* Change requests will not be considered valid without an explanation
+
+[[release-policy]]
+== Release Policy
+
+Any JanusGraph committer may propose a release.
+To propose a release, simple start a new RELEASE thread on https://groups.google.com/forum/#!forum/janusgraph-dev[janusgraph-dev] proposing the new release and requesting feedback on what should be included in the release.
+After consensus is reached the release manager will perform the following tasks:
+
+* Create a release branch so that work may continue on `master`
+* Prepare the release artifacts
+* Call a vote to approve the release on https://groups.google.com/forum/#!forum/janusgraph-dev[janusgraph-dev]
+
+Committers will be given 72 hours to review and vote on the release artifacts.
```

Comment:
This paragraph is written as a bulleted list (each sentence starts on a new line), but that's not how it will be rendered in HTML. If you'd like it to become a bulleted list, please make it so. Otherwise, please reflow the paragraph so that it reads as it will be rendered.